### ISSUE TITLE
Check the qbittorrent used port and update it if it's different to the vpn port

### ISSUE THREAD

**Issue by revers59** [SnoringDragon/gluetun-qbittorrent-port-manager#4]
Is it possible to add a check port of the current configured port of Qbittorrent and restart the script if the port is different between vpn and Qbittorrent ?

On another project I see some line to find the current used port of qbittorrent that can give some ideas :
https://github.com/soxfor/qbittorrent-natmap/blob/main/data/start.sh

When i have some update of my docker stack I see gluetun-qbittorrent-port-manager log to says succefull change port but it's not the reality.

Qbittorrent version 4.6.2

**Comment by DallasHamlin** [comment]
bump

**Comment by patrickaclark** [comment]
@DallasHamlin  @revers59

I created a fork of this project that does this/rechecks every 5 minutes as gluetun loses connection occasionally, and some VPNs vary the port on every (re)connect.  The function of the script is very different to this version as it requires/calls on the Gluetun control server API vs checking /tmp/gluetun/forwarded_port file, error checks to ensure it receives a port number.  Also added in compose dependencies and healthchecks 

Not trying to steal this creator's thunder, just based on their repos they  might be a very busy person as it's been a number of months with no response.  

[patrickaclark/gluetun-qbittorrent-port-manager](https://github.com/patrickaclark/gluetun-qbittorrent-port-manager)

**Comment by thewatermelloan** [comment]
@patrickaclark 

Any reason you shut down your port? Could you open it back up so I can use it?

**Comment by vmpr** [comment]
@patrickaclark yeah I am also missing your fork :( github was deleted and also the dockerhub image :( can you reactivate it? cheers Ringo

**Comment by patrickaclark** [comment]
@thewatermelloan @vmpr @FreekBes
I went ahead and put the github repo back.  Please feel free to fork it.  I have taken the direct image references out for my dockerhub image as I don't want to have to maintain that, but the build file is in the repo and it is easy to build and publish to your own dockerhub or other image repos.

I have moved most of my stuff to a local gitea instance, and pulled everything down one night.  I should have given a heads up, and didn't realize so many people were using my fork.  Please feel free to re-fork and continue any fixes with auth and stuff that are coming down the pipe with gluetun.  

With a lot of the uncertainty going on in the US and other life events  I haven't had a ton of time to maintain/look at this, while I'm upskilling to hopefully keep my current job or be marketable for a new one.
